http3/server.go

```diff
@@ -479,6 +496,24 @@ func (s *Server) handleConn(conn quic.Connection) error {
 		}
 	}
 
+	s.connCount.Add(1)
+	defer func() {
+		s.connCount.Add(-1)
+		s.mutex.RLock()
+		// close once
+		if s.closed && s.doneChanClosed.CompareAndSwap(false, true) {
```

Comment:
We only want to execute this when the last connection is closed, right?

Reply:
oops, forgot to check connection count 😅